app/models/author.rb

```diff
@@ -0,0 +1,25 @@
+# store author-related info for studies
+# will allow users to contact study authors directly with questions if author is listed as "corresponding"
+# all authors will default to corresponding: false
+# can also be used for search purposes
+class Author
+  include Mongoid::Document
+  include Mongoid::Timestamps
+  field :first_name, type: String
+  field :last_name, type: String
+  field :email, type: String
+  field :institution, type: String
+  field :corresponding, type: Mongoid::Boolean, default: false
```

Comment:
ORCID is the standard global identifier for academic authors.  Journals generally encourage it, and it makes tracking and uniquifying authors over time (and thus often institutions, emails, last names) feasible.

Whether now or as a separate task, supporting ORCID in our author search and/or model seems worthwhile.

```suggestion

   # ORCID, global identifier for academic authors, e.g. 0000-0001-2345-6789
  field :orcid, type: String
```